packages/shorebird_cli/lib/src/commands/init_command.dart

```diff
@@ -89,7 +100,8 @@ For more information about Shorebird, visit ${link(uri: Uri.parse('https://shore
 # This file is used to configure the Shorebird CLI.
 # Learn more at https://shorebird.dev
 
-# This is the unique identifier for your app.
+# This is the unique identifier assigned to your app.
+# It is sent to the Shorebird servers and used to identify your app.
```

Comment:
Better:
It is used by your app to request the correct patches from the Shorebird servers.

Reply:
e.g. explaining the utility to the dev/user.